src/users/users.service.ts

```diff
@@ -598,24 +598,16 @@ export class UsersService {
         let userAdmin = await this.adminUserRepository.getAdminById(adminId)
         if (userAdmin) {
             let deleteUserAdmin = await this.adminUserRepository.getAdminById(id)
+            let user = await this.usersRepository.getUserById(id)
```

Comment:
Era pra buscar o usuario depois de validar se o deleteUserAdmin não é um admin, se for fez uma consulta no banco de dados atoa.